pallets/pallet-bonded-coins/README.md

```diff
@@ -1,4 +1,137 @@
+- Create new currency pools with specified bonding curves and collateral types.
+- Mint new tokens by locking up collateral.
+- Burn tokens to release collateral.
+- Manage the lifecycle of currency pools, including refunding and destroying pools.
+
+### Rounding 
+
+Rounding issues are a problem and cannot be completely avoided due to the nature of limited resources on a computer, resulting in a lack of representation for irrational numbers. This pallet cannot guarantee mathematically exact calculations. However, it can guarantee the reproducibility of the same result based on the usage of [fixed-point][fixed-point] numbers. 
+
+
+## Key Concepts
+
+### Bonding Curve
+A bonding curve is a mathematical curve that defines the relationship between the supply of a token and its price. In this pallet, the bonding curve determines the cost of minting or burning tokens based on the current supply. The current system implements the LMSR, square root, and polynomial bonding curves.
+
+- LMSR (Logarithmic Market Scoring Rule): A market-making algorithm that adjusts prices based on the logarithm of the token supply.
+- Square Root: A bonding curve where the price is proportional to the square root of the token supply.
+- Polynomial: A bonding curve where the price is determined by a polynomial function of the token supply.
+
+More information and implementation details can be found [here][bonding-curve].
+
+### Collateral
+Collateral is an existing currency that must be locked up to mint new tokens. The amount of collateral required is determined by the bonding curve in the creation step.
+
+### Pool
+A [pool][pool-details] is a collection of bonded currencies and their associated collateral. Each pool has a unique ID and can be managed independently. Pools can be either permissioned or trustless, as specified during the creation step.
+
+- Permissioned Pools: Only the pool manager has the authority to mint and burn bonded currencies. The manager can be updated, or the privileges can be dropped entirely.
+- Trustless Pools: Any participant can mint and burn bonded currencies without requiring special permissions.
+
+## Storage Items
+
+- `Pools`: Stores details of each pool, including its bonding curve, collateral type, and current state.
+- `NextAssetId`: Tracks the next available asset ID for new currencies.
+
+## Events
+
+- `LockSet`: Emitted when a lock is set on a pool.
+- `Unlocked`: Emitted when a pool is unlocked.
+- `PoolCreated`: Emitted when a new pool is created.
+- `RefundingStarted`: Emitted when a pool starts the refunding process.
+- `DestructionStarted`: Emitted when a pool starts the destruction process.
+- `RefundComplete`: Emitted when the refund process is complete.
+- `Destroyed`: Emitted when a pool is fully destroyed.
+- `ManagerUpdated`: Emitted when the manager of a pool is updated.
+
+## Errors
+
+- `PoolUnknown`: The specified pool ID is not registered.
+- `IndexOutOfBounds`: The specified currency index is out of bounds.
```

Comment:
Can this one be a bit clearer as to what it exactly means?

Reply:
Hope this helps: [ea7228e](https://github.com/KILTprotocol/kilt-node/pull/807/commits/ea7228e002fa608ba2742d4b7442c92f73994433)